rules/brand-more-than-logo/rule.md

```diff
@@ -0,0 +1,74 @@
+<script async src="//www.instagram.com/embed.js"></script>
+**Video: Your brand is more than just a logo (40 sec)**
+
+## Key elements of brand identity
+
+### 1. Color Palette: The Emotional Trigger
```

Comment:
@bettybondoc 

- [ ] 1. Please make this and other headings with normal casing (not capitalized)